FreddysBBQ/src/OrderService/Controllers/CustomerOrderController.cs

```diff
@@ -20,32 +20,31 @@ public class CustomerOrderController : Controller
         private ILogger<CustomerOrderController> _logger;
         private IMenuService _menuService;
 
-        public CustomerOrderController(OrderContext dbContext, IMenuService menuService, ILoggerFactory fact)
+        public CustomerOrderController(IMenuService menuService, ILoggerFactory fact)
         {
-            _dbContext = dbContext;
+            //_dbContext = dbContext;
```

Comment:
I wanted to mock out any databases and communication via Eureka, but I'm too stupid to do it in .net